new_contributors/FAQ.md

```diff
@@ -6,6 +6,8 @@ We encourage all new contributors to go over our new contributor folder which ha
 ### How do I submit my 1st article?
 Visit our step-by-step guide on [how to upload your 1st EngEd article](https://github.com/section-io/engineering-education/blob/master/new_contributors/UPLOAD_INSTRUCTIONS.md#how-to-upload-an-article-for-sections-engineering-education-program) to the GitHub repo.
 
+### How do I submit a EngEd feature suggestion?
```

Comment:
Is this section missing a description?

Reply:
Yes - I just added it to not forget to add the blurb.